platform-controller/src/main/java/org/hobbit/controller/kubernetes/networkAttachmentDefinitionCustomResources/Spec.java

```diff
@@ -0,0 +1,27 @@
+package org.hobbit.controller.kubernetes.networkAttachmentDefinitionCustomResources;
```

Comment:
This is not a good package name. It is way too long and it does not abide the typical naming conventions. Please fix this.